app/forms/defence_request_form.rb

```diff
@@ -4,17 +4,21 @@ class DefenceRequestForm
 
   attr_reader :defence_request, :fields
 
-  DELEGATED_ATTRIBUTES =  :detainee_name, :gender,
-                          :detainee_age, :offences, :comments, :appropriate_adult,
-                          :appropriate_adult_reason, :house_name, :address_1, :address_2, :city, :county, :postcode,
-                          :circumstances_of_arrest, :investigating_officer_name,
+  DELEGATED_ATTRIBUTES =  :detainee_name, :gender, :offences, :comments, :appropriate_adult, :appropriate_adult_reason,
+                          :detainee_address, :circumstances_of_arrest, :investigating_officer_name,
                           :investigating_officer_shoulder_number, :investigating_officer_contact_number,
                           :fit_for_interview, :unfit_for_interview_reason, :interpreter_required, :interpreter_type
 
+  VIRTUAL_ATTRIBUTES =  :detainee_name_not_given, :detainee_address_not_given, :date_of_birth_not_given
+
```

Comment:
if these are now persisted attributes i think you can drop the VIRTUAL_ATTRIBUTES usage?